### PR TITLE
Project 2-Sandra Shumanteva Kaevska

### PR DESCRIPTION
**Pull request by sandra-shumanteva-kaevska**
I have done Blue Level, Red Level and Black Level Requirements.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,169 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Friends</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="favicon.ico" />
```
Comment (bealun) [inline code/index.html:7]: Nice detail 💫

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,169 @@
...
+          <a target="_blank" href="https://watchfriendsonline.org/"
+            >Watch online</a
+          >
+          <img
+            class="logo-tablet"
+            src="https://turbologo.com/articles/wp-content/uploads/2019/12/friends-logo-cover-1280x720.png"
+            alt="friends logo"
+          />
+          <a
+            target="_blank"
```
Comment (bealun) [inline code/index.html:52]: Good to open links in new tabs 🙌

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,169 @@
...
+          <p>
+            It’s happening: The “Friends” cast is reuniting for an exclusive
+            untitled unscripted special on HBO Max. Jennifer Aniston, Courteney
+            Cox, Lisa Kudrow, Matt LeBlanc, Matthew Perry and David Schwimmer
+            will return to the original “Friends” soundstage, Stage 24, on the
+            Warner Bros. Studio lot in Burbank to celebrate the long-running
+            series, which ended its run in 2004.
+          </p>
+        </div>
+      </section>
```
Comment (bealun) [inline code/index.html:87]: Clean structure!

File: code/index.html [hunk 4]
```diff
@@ -2,21 +2,169 @@
...
+
+      <!-- CARDS  -->
+      <section class="cards-container">
+        <div class="card">
+          <div>TV Show Friends</div>
+          <div class="card-img">
+            <img
+              class="zoom-img"
+              src="https://img.ohmymag.co.uk/article/tv/the-friends-series-lasted-10-seasons-and-was-very-successful_49dbb3f9c8027cbc65c0903e171de0f92a9b711d.jpg"
+              alt="all person of friends"
```
Comment (bealun) [inline code/index.html:97]: Perfect use of alt in the image tag, BRAVO!

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,293 @@
...
+}
+
+.nav {
+  height: 80px;
+  display: flex;
+}
+
+.nav .logo {
+  height: 100%;
+  border-radius: 15px 50px;
```
Comment (bealun) [inline code/style.css:21]: Really nice styling feature 🏆

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,293 @@
...
+  width: 150px;
+  border-radius: 15px 50px;
+}
+
+a {
+  text-decoration: none;
+  padding: 0 15px;
+}
+
+a:hover {
```
Comment (bealun) [inline code/style.css:50]: I do love me some hover effects 😻

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,293 @@
...
+/* animation */
+
+@keyframes pulse {
+  0% {
+    background-color: #bddbda;
+  }
+  100% {
+    background-color: #5cbe88;
+  }
+}
```
Comment (bealun) [inline code/style.css:149]: W O O O W!! This is so nice to see 💥 
Please continue to explore the animation with code, if you haven't already then check out: 
https://animista.net/

**Review by bealun — APPROVED** [review]
WOW! Not only one of the best tv shows in world but also this great structure of code that you've written 👏 It's an absolute thrill to look at your site. The animation looks so good, the half rounded corners and the hover effects. I can tell you've put a lot of thought and work into this projects because it really shows. 
Please celebrate your amazing work and be super proud of yourself, you did good!! 🎉

**Comment by sandra-shumanteva-kaevska** [comment]
Wow thanks a lot for the nice review. I didn't expect this. Today I finished my project and it was very intense week for me. I was liltle down with my emotions. This gives me big motivation for next project. I am very happy and I will celebrate this. :*

**Pull request closed without merging**